File: python/Array/3052.py
import sys

if __name__ == "__main__":
    
    count=0
    arr = [0 for i in range(42)]  # 42개 까지 요소를 0으로 초기화
    
    for val in range(10):
        a=int(input())
        if(arr[a%42]==0):
            arr[a%42]=1
        
    
    for val in arr:
        count+=val
    
    print(count)
    

File: Go/Array/3052.go
package main

import (
	"bufio"
	"fmt"
	"os"
)

func main() {
	var a, count int
	reader := bufio.NewReader(os.Stdin)
	var inputs [42]int // 기본값이 0 인 List

	for i := 0; i < 10; i++ {
		fmt.Fscan(reader, &a)
		if inputs[a%42] == 0 {
			inputs[a%42] = 1
		}
	}

	for _, val := range inputs {
		count += val
	}
	fmt.Print(count)
}
